src/article.twig

```diff
@@ -0,0 +1,28 @@
+{# article.html #}
+{% extends "_pages/page_layout.twig" %}
+<head>
+    {% block head %}
+        <link rel="stylesheet" href="css/article.css"/>
+    {% endblock %}
```

Comment:
You replaced the block name in the page_layout template but did not do the adjustment here, please replace `head` with `additional_styles`